### DISCUSSION TITLE
Taro H5 服务端渲染（SSR）插件发布正式版！🎉

### DISCUSSION THREAD

**Discussion by SyMind** [NervJS/taro#11996] [生态] [answered]
目前发布 1.0.0 版本，意味着插件逐渐趋于稳定，欢迎大家使用！

插件介绍：让 Taro H5 支持 Pre-rendering、SSR 和 ISR，提升首屏速度🚀，利于 SEO🔍。

项目地址：https://github.com/NervJS/tarojs-plugin-platform-nextjs

请 Star 🌟 这个项目来表达你的喜爱 ❤️ 和支持。

**Comment by airqj** [comment, marked as answer]
已star
  👍🏻
